test/integration/liquidation.test.js

```diff
@@ -105,6 +138,188 @@ describe("Integration -- liquidation", () => {
         bondedAmount
       )
     })
+
+    it("should transfer collateral tokens to bidder", async () => {
+      expect(await collateralToken.balanceOf(bidder.address)).to.be.closeTo(
+        to1e18(15),
+        precision
+      )
+      expect(await collateralToken.balanceOf(assetPool.address)).to.be.closeTo(
+        to1e18(85),
+        precision
+      )
+    })
+  })
+
+  describe("when auction fully filled after auction length elapsed", () => {
+    let auction
+    let tx
+
+    beforeEach(async () => {
+      auction = await prepareAuction()
+      increaseTime(auctionLength) // Wait the whole auction length
+      tx = await auction.connect(bidder).takeOffer(lotSize)
+    })
+
+    it("should close auction", async () => {
+      expect(await auction.isOpen()).to.be.false
+    })
+
+    it("should liquidate the deposit", async () => {
+      // Auction bidder has spend their TBTC
+      expect(await tbtcToken.balanceOf(bidder.address)).to.equal(0)
+      // Deposit has been liquidated
+      expect(await tbtcDeposit.currentState()).to.equal(11) // LIQUIDATED
+    })
+
+    it("should swap signer bonds", async () => {
+      await expect(tx).to.changeEtherBalance(riskManagerV1, 0)
+      await expect(tx).to.changeEtherBalance(
+        signerBondsSwapStrategy,
+        bondedAmount
+      )
+    })
+
+    it("should transfer collateral tokens to bidder", async () => {
+      expect(await collateralToken.balanceOf(bidder.address)).to.be.equal(
+        to1e18(100)
+      )
+      expect(await collateralToken.balanceOf(assetPool.address)).to.be.equal(0)
+    })
+  })
+
+  describe("when auction fully filled with multiple take offers", () => {
+    let auction
+    let tx
+
+    beforeEach(async () => {
+      auction = await prepareAuction()
+      await tbtcToken.connect(otherBidder).approve(auction.address, lotSize)
+
+      // Step 1: wait 5% of auction length (4320s) and take 40% of amount
+      // (4 * 10^18) on auction with bidder (amountToTransfer)
+
+      // Parameters needed for calculation:
+      // auctionLength = 86400
+      // amountOutstanding = 10 * 10^18 (amount that is left on auction)
+      // startTimeOffset = 0 (initial value)
+      // velocityPoolDepletingRate = 1.0 (initial value)
+      // amountOnOffer = (timeSinceAuctionStart - startTimeOffset)
+      //   * velocityPoolDepletingRate / auctionLength =
+      //   (4320 - 0) * 1.0 / 86400 = 0.05
+      // portionToSeize = amountOnOffer * amountToTransfer / amountOutstanding =
+      //   0.05 * (4 * 10^18) / (10 * 10^18 ) = 0.02
+      // amountInAssetPool = 100 * 10^18
+      // amountToSeize = amountInAssetPool * portionToSeize =
+      //   (100 * 10^18) * 0.02 = 2 * 10^18
+      // Bidder should receive (2 * 10^18) of collateral tokens
```

Comment:
Posting all these calculations here feels like an overkill, but I wanted to show how we arrived with the values of collateral tokens seized from asset pool that we expect to get in test cases.

Reply:
> Alternatively, we could just check that after the auction was fully filled, the total number of collateral tokens the bidders received is equal to the initial amount of collateral tokens in the asset pool.

Plus, we could check that what they received is what the auction offered, especially if bidding through `AuctionBidder.takeOfferWithMin`. 

I do agree those calculations are overkill, I would leave them for unit tests.